data/en/bots/435454917616861197.md

```diff
@@ -0,0 +1,22 @@
+---
+avatar: >-
+  https://images.discordapp.net/avatars/435454917616861197/8a350b3ded637ad480cfce7f49773afd.png
+cover: null
+custom_path: null
+description: >-
+  You can create your server with this bot in one command. All you have to do is
+  make sure the bot has all permissions and you do also then do r!setup and
+  follow the steps.
+github:
+  owner: squazy
```

Comment:
imo should you not add yourself for GitHub, if you don't have a repository to share.